package.json

```diff
@@ -29,6 +29,7 @@
     "lint": "npm-run-all lint:*",
     "lint:css": "stylelint 'webextension/skin/*.css'",
     "lint:js": "eslint webextension --ext=js,json,jsm",
+    "postinstall": "node_modules/shield-studies-addon-utils/bin/copyStudyUtils.js webextension/privileged",
```

Comment:
Nit: Can be shortened to `copyStudyUtils webextension/privileged"

Reply:
Fix in PR https://github.com/punamdahiya/shield-cloudstorage/pull/1/files

This works for me once I update Path in bash_profile  to $PATH:$(npm bin)